.github/workflows/android.yml

```
@@ -0,0 +1,215 @@
+    environment: SAKSHAM_STAG
+
+    strategy:
+      matrix:
+        variant: [ SakshamStagDebug ]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up JDK
+        uses: actions/setup-java@v2
```

actionlint (1.7.4)

the runner of "actions/setup-java@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)